conference.js

```diff
@@ -756,6 +742,20 @@ export default {
     isLocalId(id) {
         return this.getMyUserId() === id;
     },
+
+    /**
+     * Tells whether the local video is muted or not.
+     * @return {boolean}
+     */
+    isVideoMuted() {
+        // If the tracks are not ready, read from base/media state
+        return this._localTracksInitialized
+            ? isLocalTrackMuted(
```

Comment:
nope, this.localVideo can remain undefined even after local tracks have been initialised (startWithVideoMuted). When this flag turns to true we are sure that this.localTrack is either there or not and can make decision to eventually create one unmute.